src/Support/Field.php

```diff
@@ -94,7 +99,38 @@ public function getValidator(array $args, array $rules): ValidatorContract
         return Validator::make($args, $rules, $messages);
     }
 
-    protected function getResolver(): ?Closure
+    protected function getResolver() : ?Closure
```

Comment:
```suggestion
    protected function getResolver(): ?Closure
```

Guess that's a slap for php-cs-fixer, need to look into this